.jshintrc

```diff
@@ -2,13 +2,13 @@
     "bitwise": false,
     "camelcase": false,
     "curly": true,
-    "eqeqeq": true,
+    "eqeqeq": false,
```

Comment:
These settings should be reverted to our project defaults - in particular the use of the === operator correponds to a JS best practice http://stackoverflow.com/questions/359494/does-it-matter-which-equals-operator-vs-i-use-in-javascript-comparisons#answer-359509